integration-tests/pom.xml

```diff
@@ -36,6 +36,27 @@
         </dependency>
     </dependencies>
     <build>
+        <pluginManagement>
```

Comment:
I added this to the IT so I could build a real docker image and test it running.